app/views/layouts/base.html.erb

```diff
@@ -15,6 +15,7 @@
     <%= yield(:stylesheets) %>
 
     <%= csrf_meta_tags %>
+    <%= javascript_include_tag *webpack_asset_paths('vendor', :extension => 'js'), "data-turbolinks-track" => true %>
```

Comment:
I see it's mostly React (the library itself), however it doesn't seem to get my `../foreman_ansible/webpack/index.js` content in it.. https://gist.github.com/dLobatog/96f33a68be7fdd92ba79e8d5d6219496 is what I see